spec.go

```diff
@@ -37,13 +37,21 @@ func (c *SpecController) Show(ctx *app.ShowSpecContext) error {
 	if len(elems) < 3 {
 		return fmt.Errorf("invalid package path %s", packagePath)
 	}
+	var branch string
+	last := elems[len(elems)-1]
+	if strings.Contains(last, ":") {
```

Comment:
We could use `@` instead of `:` to be consistent with other tools like [govendor](https://github.com/kardianos/govendor).